finalPrep/modZeroHero.js

```diff
@@ -55,17 +99,51 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 //   - citiesDestroyed (array)
 //   - luckyNumbers (array)
 //   - address (object with following key/values: number , street , state, zip)
-
+var scaryMonster = {
+  name: "Sleepless Toddler",
+  smell: "poopy",
+  weight: 23,
+  citiesDestroyed: ["Denver", "Pueblo", "Greeley"],
+  luckynumbers: [6, 24, 19],
+  address: {
+    number: 8327,
+    street: "Ames Way",
+    state: "Colorado",
+    zip: "80003",
+  }
+}
 
 // Create a new class called SuperHero
 // - Your class should have the following DYNAMIC values
-//   - name 
+//   - name
 //   - superpower
-//   - age 
+//   - age
 // - Your class should have the following STATIC values
 //   - archNemesis, assigned to "The Syntax Error"
 //   - powerLevel = 100
-//   - energyLevel = 50 
+//   - energyLevel = 50
+class SuperHero {
+  constructor(name, superpower, age){
+    this.name = name;
+    this.power = superpower;
+    this.age = age;
+    this.archNemesis = "The Syntax Error";
+    this.powerLevel = 100;
+    this.energyLevel = 50;
+  }
+  sayName() {
+    this.name;
```

Comment:
You still need the `console.log` here, the instructions ask you to:

> Create the following class methods  - sayName, should print the hero's name to the console

You had the `console.log` correct initially, however the string you passed in was not the "hero's name" which is a dynamic value stored in the `this.name` instance variable. So what I expected to see here is:

```
sayName() {
    console.log(this.name)
}
```

Reply:
OOOOHHHH!! That makes sense, now. Thanks for modeling that.